Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM rust
+
+RUN cargo install cargo-watch cargo-deb
```

Comment:
I wouldn't provide any distro-specific stuff.

Reply:
`cargo-deb` is necessary to build the `deb` packages, which might be good to have it accessible.

Reply:
Yeah I know what it would be for but it's distro-specific and hence not something I'd want here.

Is building deb natively harder?

Reply:
What do you mean with `natively`? Without using `cargo-deb`?

Reply:
I mean w/o using docker or any containers, like I normal develop. :)